backend/src/test/java/heartbeat/service/report/ReportServiceTest.java

```diff
@@ -104,4 +101,25 @@ void ShouldInitializeMetricsDataCompletedInHandlerWithPreOneWhenPreOneExisted()
 		verify(generateReporterService, never()).generateDoraReport(request);
 	}
 
+	@Test
+	void ShouldInitializeMetricsDataCompletedInHandlerWhenRequestMetricsExist() throws InterruptedException {
+		GenerateReportRequest request = GenerateReportRequest.builder()
+			.csvTimeStamp("csvTimeStamp")
+			.metrics(List.of("velocity", "lead time for changes"))
```

Comment:
do we have enum for `("velocity", "lead time for changes"`

Reply:
done